src/remediations/fifi.js

```diff
@@ -41,17 +49,49 @@ exports.getSatelliteId = function (facts) {
     return null;
 };
 
+function defineRHCEnabledExecutor (satellites) {
+    const lastElement = _.last(satellites);
```

Comment:
how confident are you that the "bucket of non-satellite systems" is always going to be the last element in the list?